iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/parser/ASTVisitor.java

```diff
@@ -3313,6 +3314,12 @@ public Statement visitCreateSchemaTemplate(IoTDBSqlParser.CreateSchemaTemplateCo
       List<CompressionType> compressors = new ArrayList<>();
       for (IoTDBSqlParser.TemplateMeasurementClauseContext templateClauseContext :
           ctx.templateMeasurementClause()) {
+        if (templateClauseContext
+            .nodeNameWithoutWildcard()
+            .getText()
+            .equalsIgnoreCase(STRING_NULL)) {
+          throw new SemanticException("Device Template: measurement's name can not be 'null'.");
```

Comment:
Refactor the error msg to "The name of a measurement in schema template shall not be null."